packages/twenty-front/src/modules/workflow/workflow-actions/components/WorkflowEditActionFormSendEmail.tsx

```diff
@@ -79,14 +76,13 @@ export const WorkflowEditActionFormSendEmail = ({
   };
 
   useEffect(() => {
-    form.setValue(
-      'connectedAccountId',
-      action.settings.input.connectedAccountId ?? '',
-    );
-    form.setValue('email', action.settings.input.email ?? '');
-    form.setValue('subject', action.settings.input.subject ?? '');
-    form.setValue('body', action.settings.input.body ?? '');
-  }, [action.settings, form]);
+    setFormData({
+      connectedAccountId: action.settings.input.connectedAccountId,
+      email: action.settings.input.email,
+      subject: action.settings.input.subject ?? '',
+      body: action.settings.input.body ?? '',
+    });
+  }, [action.settings.input]);
```

Comment:
Honestly, I'm not sure if we need this useEffect. If the server takes time to respond, the form's state might be corrupted.